tensorflow/core/ops/math_ops.cc

```diff
@@ -167,7 +167,7 @@ REGISTER_OP("_MklBatchMatMul")
     .Input("x: T")
```

Comment:
Maybe these ops should be moved to a separate file, like the was done for nn ops?

Reply:
Is it okay if I do it in a separate PR? Because it would require more changes and additional testing as well. Please let me know. Thanks.